src/tool-core/models/tool-core/architecture/interfaces/Message/Message.xtuml

```diff
@@ -14,12 +14,16 @@ INSERT INTO O_TFR
 	'',
 	"ba5eda7a-def5-0000-0000-000000000000",
 	1,
-	'// Generate the interface methods to create Port Messages.
-// An Interface mark may specify message parameters passed as JSON strings, rather than indexed array.
-// Parameter-handling for both mechanisms are built; one is then chosen based on that marked value.
+	'// Render the Interface file methods which create Port Messages.
+// Note that any parameters of type inst_ref or inst_ref_set must be fully qualified as component-independent.
```

Comment:
In this comment, please explain why the fully-qualified name of the component-independent type is used instead of requesting a type reference for it with a goal of producing an import statement for the component-independent type.